tensorpack/train/trainers.py

```diff
@@ -371,37 +371,63 @@ class HorovodTrainer(SingleCostTrainer):
            `ResNet-Horovod <https://github.com/tensorpack/benchmarks/tree/master/ResNet-Horovod>`_
            for a full example which has handled these common issues.
            This example can train ImageNet in roughly an hour following the paper's setup.
+    Attributes:
+        BROADCAST_EVERY_EPOCH (bool):
+            Whether to broadcast the variables every epoch.
+            Theoretically this is a no-op (because the variables
+            are supposed to be in-sync).
+            But this cheap operation may help prevent
+            certain numerical issues in practice.
     """
-    def __init__(self, average=True):
+
+    BROADCAST_EVERY_EPOCH = False
```

Comment:
Did you consider making this configurable?

Reply:
No. One reason is that this flag is inside tensorpack (There are lots of hard-coded flags in tensorpack). Also if it works fine without broadcast, we won't want to switch it back.